src/code/z_play.c

```diff
@@ -483,7 +483,8 @@ void Play_Init(GameState* thisx) {
     Camera_InitDataUsingPlayer(&this->mainCamera, player);
     Camera_RequestMode(&this->mainCamera, CAM_MODE_NORMAL);
 
-    playerStartBgCamIndex = PARAMS_GET_U(player->actor.params, 0, 8);
+    playerStartBgCamIndex = PLAYER_GET_START_BG_CAM_INDEX(&player->actor);
```

Comment:
https://github.com/zeldaret/oot/blob/ce924d40d950f047650e4a2c42e75ab164990c27/include/z64player.h#L18

nit: I see that it has changed access from `PARAMS_GET_U` to `PARAMS_GET_S`, if that matters

Reply:
I thought this would fix the problem mentioned in a review comment above, but unfortunately it does not.
Ill change it back anyway I guess